integration_tests/src/main.rs

```diff
@@ -9,13 +9,31 @@ async fn insert_schema(client: &tokio_postgres::Client, filename: &str) -> Resul
     client.batch_execute(schema.as_str()).await
 }
 
-async fn create_data_partition(client: &tokio_postgres::Client) -> Result<(), Error> {
-    // TODO: add multiple partitions?
```

Comment:
Idk if it's intentional that you dropped this todo

Reply:
I guess there isn't really a point in having multiple partitions here unless we want to perform specific tests that make use of them. But yes, I think it's probably worth keeping the TODO